### PR TITLE
Feature/clean up

### PR DESCRIPTION
**Pull request by fryette**

### CODE REVIEW

File: ecommers/lib/core/provider_models/home_provider_model.dart [hunk 1]
```diff
@@ -33,24 +34,24 @@ class HomeProviderModel extends ProviderModelBase with ItemsLoadingNotifier {
   }
 
   Future _fetchCategories() async {
-    _categoryList = await services.categoryService.fetchCategoryList();
```
Comment (amarchenko25) [inline ecommers/lib/core/provider_models/home_provider_model.dart:36]: the reason why I added services alias to the package is that paginator field from the dependency_services conflicts with the paginator from ItemsLoadingNotifier:)

File: ecommers/lib/core/provider_models/products_grid_provder_model.dart [hunk 2]
```diff
@@ -72,7 +68,7 @@ class ProductsGridProviderModel extends ProviderModelBase
   }
 
   Future<List<Product>> _fetchMoreProducts(int from, int to) {
-    return services.productService.fetchProducts(
```
Comment (amarchenko25) [inline ecommers/lib/core/provider_models/products_grid_provder_model.dart:75]: the reason why I added services alias to the package is that paginator field from the dependency_services conflicts with the paginator from ItemsLoadingNotifier:)

**Review by amarchenko25 — DISMISSED** [review]

File: ecommers/lib/core/provider_models/home_provider_model.dart (hunk 1 above)
Comment (fryette) [reply ecommers/lib/core/provider_models/home_provider_model.dart:36]: Seems now everything is ok, am I right?
Also maybe better to rename it accordingly?

File: ecommers/lib/core/provider_models/products_grid_provder_model.dart (hunk 2 above)
Comment (fryette) [reply ecommers/lib/core/provider_models/products_grid_provder_model.dart:75]: Seems now everything is ok, am I right?
Also maybe better to rename it accordingly?

**Review by amarchenko25 — APPROVED** [review]

**Pull request merged**